eventRegistration/models.py

```diff
@@ -1,11 +0,0 @@
-from django.db import models
-
-from events.models import Event
-from patrons.models import Patron
-
-
-# Create your models here.
-class eventRegistration(models.Model):
```

Comment:
need to add class Meta
add relevant helping + error messages
__str__ function is not declared
first character of the class name should be capital